### PR TITLE
add descriptions to PodCondition struct tags

### PR DESCRIPTION
**Pull request by mikedanese**
Per #4542
/cc @bgrant0607

### CODE REVIEW

File: pkg/api/v1beta1/types.go [hunk 1]
```diff
@@ -445,8 +445,8 @@ const (
 
 // TODO: add LastTransitionTime, Reason, Message to match NodeCondition api.
 type PodCondition struct {
-	Kind   PodConditionKind `json:"kind"`
-	Status ConditionStatus  `json:"status"`
+	Kind   PodConditionKind `json:"kind" description:"kind of the condition, currently only ready"`
```
Comment (bgrant0607) [inline pkg/api/v1beta1/types.go]: Please make the capitalization match the constants.

**Comment by bgrant0607** [comment]
Thanks much for the quick PR.

File: pkg/api/v1beta1/types.go (hunk 1 above)
Comment (mikedanese) [inline pkg/api/v1beta1/types.go]: Fixed and for node condition too.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**